src/utils.js

```diff
@@ -59,7 +60,7 @@ async function getConfig(file: string): Promise<ZelConfig> {
  * @param {string} file
  * @return {Promise<T>}
  */
-async function sync(repo: string, branch: string, file: string): Promise<T> {
+async function sync<T>(repo: string, branch: string, file: string): Promise<T> {
```

Comment:
added missing generic `T` in type parameter list.

**Side note:** We may want to remove `bluebird` since there's been some [incompatibilities](https://github.com/flowtype/flow-typed/issues/504) with typings for `await`ing a `Bluebird$Promise` type.

There are [workarounds](https://github.com/flowtype/flow-typed/issues/504#issuecomment-268795292) but it adds additional unnecessary complexity.